condicionales/comparatotAnos-2-6.js

```diff
@@ -0,0 +1,21 @@
+// Escriba un programa que pida un año y que escriba si es bisiesto o no.Se recuerda que los años bisiestos son múltiplos de 4, pero los múltiplos de 100 no lo son, aunque los múltiplos de 400 sí.
+const prompt = require('prompt-sync')();
```

Comment:
in javascript is possible to work with both ' and " but you have to choose which one you are going to choose or the one that uses the project. Here we use "